.travis.yml

```diff
@@ -3,34 +3,22 @@ language: node_js
 node_js:
   - "node"
 
-env:
-  global:
-    - GH_REF: github.com/w3c/aria-practices.git
-
 git:
   depth: 3
 
 branches:
```

Comment:
Just realized with this approach, this filter can be removed so PRs outside `master` can be built. I was scratching my head to why 440 wasn't running :stuck_out_tongue: